### DISCUSSION TITLE
Where do you learn aos

### DISCUSSION THREAD

**Discussion by shape1541** [community/community#19602] [Other Feature Feedback, Questions, & Ideas] [answered]
Where do you learn aos library

**Comment by melvincwng** [comment, marked as answer]
@shape1541, just to clarify, for aos - do you mean "Animate On Scroll"?

If that's the case, consider using this easy-to-use aos library created by Michal over [here](https://github.com/michalsnik/aos).

Hope this is helpful.

**Reply by melvincwng** [reply]
@shape1541, if this reply has helped you, please kindly 'mark as answer' and follow me on GitHub.

Appreciate the support given ^^ in advance.

Thank you, and take care 😄
